draftjs_exporter/wrapper_state.py

```diff
@@ -128,7 +128,16 @@ def element_for(self, block):
         options = Options.for_block(self.block_map, type_)
 
         # Make an element from the options specified in the block map.
-        elt = DOM.create_element(options.element[0], options.element[1])
+        tag = options.element[0]
+        attributes = options.element[1]
+
+        if (hasattr(tag, 'render')):
```

Comment:
I don't think you need this – `DOM.create_element` will directly call `render` if it detects that `tag` has a render method or similar.